src/Storage/migrations/4_9_0.ts

```diff
@@ -18,3 +21,7 @@ export const migrateAuthSetting = async (storage: Storage) => {
     return storage.setItem(SettingsStorageKeys.Auth, JSON.stringify(AUTH_WITH_PIN))
   }
 }
+
+export const to4_9_0 = async () => migrateAuthSetting(storage)
```

Comment:
is this necessary?
or maybe?
`export const to_4_9_0 = migrateAuthSetting`

Reply:
```suggestion
export const to4_9_0 = migrateAuthSetting
```

Reply:
I thinking about a interface for all migrations like `migrate` , going with the suggestion for now